schemas/test/en/test_numbers.json

```diff
@@ -205,7 +205,7 @@
                                     },
                                     {
                                         "id": "test-min-exclusive",
-                                        "label": "Min Exclusive Test (124 to 9,999,999,999 - 123 Exclusive)",
+                                        "label": "Min Exclusive Test (124 to 9,999,999,999,999 - 123 Exclusive)",
```

Comment:
Same as prev comment - enforced upper limit here is 999,999,999,999,999, not 9,999,999,999,999 as stated in test description.

Reply:
Updated 👍